README.md

```diff
@@ -535,8 +535,9 @@ final class TestExample extends TestCase
     {
         $message1 = new Message();
         $message2 = new Message();
-        $messageRepository = $this->createStub(MessageRepositoryInterface::class);
-        $messageRepository->method('getAll')->willReturn([$message1, $message2]);
+        $messageRepository = new InMemoryMessageRepository();
```

Comment:
I think that example with phpunit api is also good, maybe there could be two examples of stub, wdyt?

Reply:
This is a good question. I wonder what are pros and cons of using stub vs InMemory are.

Reply:
@sargath @speccode thanks for the comments 👍 
I've just added an explanation of why I think that using own classes is better, let me know what you think